src/core/scene/a-scene.js

```diff
@@ -721,9 +772,23 @@ class AScene extends AEntity {
     var systems = this.systems;
 
     // Components.
-    for (i = 0; i < this.behaviors.tock.length; i++) {
-      if (!this.behaviors.tock[i].el.isPlaying) { continue; }
-      this.behaviors.tock[i].tock(time, timeDelta, camera);
+    for (var c = 0; c < this.componentOrder.length; c++) {
```

Comment:
wonder if this code and above can be factored out into a function. `callBehavior('tick') / callBehavior ('tock')`